### PR TITLE
[#11][UI] As a user, I can see placeholder when the survey list is empty

### LINKED ISSUE #11: [UI] As a user, I can see placeholder when the survey list is empty
## Why
The authenticated user can visit the home page to see the survey dashboard. In the dashboard, the user can see the current date and list of available surveys and select which survey the user wants to take. The list of surveys is cached when the user first visits the dashboard, and the cache gets refreshed every time the user refreshes the page.

## Acceptance Criteria
Display placeholder when the survey list is empty.
image

### PR DESCRIPTION
**Pull request by hanam1ni**
- Closes #11 

## What happened 👀

Add placeholder to display when survey list is empty
 
## Proof Of Work 📹

image

### CODE REVIEW

**Review by tyrro — APPROVED** [review]
not sure though, should we change the background to dark to keep in sync with Figma? the rests LGTM!

File: src/pages/index.page.tsx [hunk 1]
```diff
@@ -21,6 +22,18 @@ const Home: NextPage = () => {
               </div>
               <div className="text-4xl font-extrabold">Today</div>
             </section>
+            <section className="mt-36 flex flex-col items-center">
+              <Image
+                width={64}
+                height={64}
+                src="/icon/smiley.svg"
+                alt="placeholder icon"
```
Comment (hoangmirs) [inline src/pages/index.page.tsx:30]: Should it be `Smiley icon`? 😄

**Review by hoangmirs — APPROVED** [review]

File: src/pages/index.page.tsx [hunk 2]
```diff
@@ -21,6 +22,18 @@ const Home: NextPage = () => {
...
+              />
+              <h2 className="mt-8 text-2xl font-extrabold text-center">
```
Comment (carryall) [inline src/pages/index.page.tsx:32]: Should we adjust the style a bit as the font size and the `line-height` is not the same as on the design

The design | Current style
--- | ---
Screen Shot 2565-05-12 at 11 07 23 | Screen Shot 2565-05-12 at 11 08 02

Comment (hanam1ni) [reply src/pages/index.page.tsx]: Updated text size and line height 
Screen Shot 2565-05-12 at 16 41 08

**Comment by hanam1ni** [comment]
@tyrro Add functionality to dynamically set the background image as we will need this in the future implementation. For placeholder, I set background to black as you mentioned 👍
Screen Shot 2565-05-12 at 17 56 48

**Review by carryall — APPROVED** [review]

**Pull request merged**